Compile errors on armhf

Trying to build on a Raspberry Pi 3B+, running up to date Debian Buster (Raspbian), clang-8 from Debian backports.

```
$ uname -a
Linux raspberrypi 4.19.118-v7+ #1311 SMP Mon Apr 27 14:21:24 BST 2020 armv7l GNU/Linux
```

Environment:
```
export CC=clang-8
export CXX=clang++-8
export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
export RUNTIME_VERSION=gnustep-2.0
export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
export LD=/usr/bin/ld.gold
export LDFLAGS="-fuse-ld=/usr/bin/ld.gold -L/usr/local/lib -I/usr/local/include"
export OBJCFLAGS="-fblocks"
```

libobjc2 configured with:
```
cmake .. -DCMAKE_C_COMPILER=${CC} \
-DCMAKE_CXX_COMPILER=${CXX} \
-DCMAKE_BUILD_TYPE=Release \
-DUSE_GOLD_LINKER=YES
```

Throws this compilation error 8 times:
```
pi@raspberrypi:~/Documents/gnustep-build/debian-10-clang-8.0/libobjc2/build $ make -j2
[  1%] Building C object CMakeFiles/objc.dir/eh_personality.c.o
[  1%] Building C object CMakeFiles/objc-static.dir/eh_personality.c.o

/home/pi/Documents/gnustep-build/debian-10-clang-8.0/libobjc2/unwind-arm.h:208:54: note: expanded from macro 'COPY_EXCEPTION'
  memcpy((dst)->cleanup_cache, (src)->cleanup_cache, sizeof(struct cleanup_cache)); \
                                                     ^     ~~~~~~~~~~~~~~~~~~~~~~
/home/pi/Documents/gnustep-build/debian-10-clang-8.0/libobjc2/eh_personality.c:571:3: note: forward declaration of 'struct cleanup_cache'
                COPY_EXCEPTION(ex->cxx_exception, exceptionObject)
                ^
/home/pi/Documents/gnustep-build/debian-10-clang-8.0/libobjc2/unwind-arm.h:208:68: note: expanded from macro 'COPY_EXCEPTION'
  memcpy((dst)->cleanup_cache, (src)->cleanup_cache, sizeof(struct cleanup_cache)); \
                                                                   ^
/home/pi/Documents/gnustep-build/debian-10-clang-8.0/libobjc2/eh_personality.c:575:3: error: invalid application of 'sizeof' to an incomplete type
      'struct pr_cache'
                COPY_EXCEPTION(exceptionObject, ex->cxx_exception)
```

--- a/unwind-arm.h
+++ b/unwind-arm.h
@@ -203,7 +203,7 @@ _Unwind_Reason_Code name(_Unwind_State state,\
 #define CALL_PERSONALITY_FUNCTION(name) name(state,exceptionObject,context)
 
 #define COPY_EXCEPTION(dst, src) \
-  memcpy((dst)->unwinder_cache, (src)->unwinder_cache, sizeof(struct unwinder_cache)); \
-  memcpy((dst)->barrier_cache, (src)->barrier_cache, sizeof(struct barrier_cache)); \
-  memcpy((dst)->cleanup_cache, (src)->cleanup_cache, sizeof(struct cleanup_cache)); \
-  memcpy((dst)->pr_cache, (src)->pr_cache, sizeof(struct pr_cache));
+  (dst)->unwinder_cache = (src)->unwinder_cache; \
+  (dst)->barrier_cache = (src)->barrier_cache; \
+  (dst)->cleanup_cache = (src)->cleanup_cache; \
+  (dst)->pr_cache = (src)->pr_cache;

